pl_examples/intel_compressor/config/quantization.yaml

```diff
@@ -0,0 +1,20 @@
+version: 1.0
+
+model:
+  name: imagenet
+  framework: pytorch_fx                              # supported values are pytorch, pytorch_fx, pytorch_ipex...and so on.
+
+device: cpu
+
+quantization:
```

Comment:
In the readme, more notes on each quantization method would be appreciated.

Reply:
done